templates/docs/settings/spacing-settings.md

```diff
@@ -7,70 +7,60 @@ context:
 # Spacing
 
 <hr>
+Spacing in Vanilla is controlled via a set of variables. There are two kinds of variables - nudges, which keep text aligned to the baseline grid, and multiples of the spacing unit, which define vertical and horizontal spacing.
```

Comment:
Any reason why you list "nudges" always before standard 'multiples' spacing?

I think it would be clearer to do the opposite. The default (and easier to understand) are the spacing unit based variables, only if something doesn't align to baseline it uses nudges.
So in case of this sentence:

```suggestion
Spacing in Vanilla is controlled via a set of variables. There are two kinds of variables - multiples of the spacing unit, which define vertical and horizontal spacing. and nudges, which keep text aligned to the baseline grid.
```

Reply:
Precisely because they are more likely to be overlooked - especially if left after a second "and". they are also the firsrt things thst need to be set right, otherwise  nothing would align to the baseline grid, and the multiples would be pointless.